test/kubernetes/e2e/features/server_tls/types.go

```diff
@@ -19,11 +21,17 @@ var (
 	tlsSecret1Manifest      = func() ([]byte, error) { return manifestFromFile("tls-secret-1.yaml") }
 	tlsSecret2Manifest      = func() ([]byte, error) { return manifestFromFile("tls-secret-2.yaml") }
 	tlsSecretWithCaManifest = func() ([]byte, error) { return manifestFromFile("tls-secret-with-ca.yaml") }
+
 	vs1Manifest             = func() ([]byte, error) { return manifestFromFile("edge/vs-1.yaml") }
 	vs2Manifest             = func() ([]byte, error) { return manifestFromFile("edge/vs-2.yaml") }
 	vsWithOneWayManifest    = func() ([]byte, error) { return manifestFromFile("edge/vs-with-oneway.yaml") }
 	vsWithoutOneWayManifest = func() ([]byte, error) { return manifestFromFile("edge/vs-without-oneway.yaml") }
 
+	gatewayManifest   = func() ([]byte, error) { return manifestFromFile("k8s/gateway.yaml") }
+	httprouteManifest = func() ([]byte, error) { return manifestFromFile("k8s/httproute.yaml") }
```

Comment:
Just being extremely nitpicky: let's match the format of other word breaks in this section
```suggestion
	httpRouteManifest = func() ([]byte, error) { return manifestFromFile("k8s/http-route.yaml") }
```